cmd/rootCmd.go

```diff
@@ -96,26 +98,26 @@ func Baseline(ctx context.Context, output string) {
 	}
 	fmt.Println("Available Storage Provisioners:")
 	fmt.Println()
-	time.Sleep(500 * time.Millisecond)
+	time.Sleep(500 * time.Millisecond) // Added to introduce lag.
 	for _, provisioner := range provisionerList {
 		provisioner.Print()
 		fmt.Println()
 		time.Sleep(500 * time.Millisecond)
 	}
```

Comment:
Why the delays?

Reply:
I initially introduced them so that it doesn't just puke out a bunch of information. Makes it easier to track with eyes. I intentionally commented there to think of an alternative later. Cosmetic